contributed_definitions/nyaml/NXexternal_stimulus.yaml

```diff
@@ -0,0 +1,43 @@
+category: base
+doc: |
+  Parameters for controlling external simuli during an experiment.
+type: group
+NXexternal_stimulus(NXenvironment):
+  name:
+    doc: |
+      Apparatus identification code/model number; e.g. OC100 011
+  short_name:
+    doc: |
+      Alternative short name, perhaps for dashboard display like a present Seblock name
+  type:
```

Comment:
would it make sense to move these fields to NXfabrication and then just hook in an instance of NXfabrication